src/hooks/useEditNote.test.js

```diff
@@ -0,0 +1,35 @@
+import "@testing-library/jest-dom";
+import { mockError, mockResponseNote } from "../utils/mocks/notesMocks";
+import { useEditNote } from "./useEditNote";
+
+beforeEach(() => {
+  fetch.resetMocks();
+});
+
+describe("useAddNote hook", () => {
```

Comment:
Remember to check the description of the test to avoid confusion. This one is for "useEditNote"